ghost/core/test/unit/shared/config/helpers.test.js

```diff
@@ -56,29 +53,3 @@ describe('vhost utils', function () {
         });
     });
 });
-
-describe('getContentPath', function () {
```

Comment:
No longer needed as we are reverting the changes added [here](https://github.com/TryGhost/Ghost/pull/21857)